homeassistant/components/sensor/transmission.py

```diff
@@ -32,13 +31,15 @@
     'paused_torrents': ['Paused Torrents', None],
     'total_torrents': ['Total Torrents', None],
     'upload_speed': ['Up Speed', 'MB/s'],
+    'completed_torrents': ['Completed Torrents', None],
+    'started_torrents': ['Started Torrents', None],
 }
 
 SCAN_INTERVAL = timedelta(minutes=2)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
The platform schema should be removed if we load the platform via discovery from the component.